src/assets/we-are-using/zeoagency.svg

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 21.1.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 viewBox="0 0 1635.9 627.7" style="enable-background:new 0 0 1635.9 627.7;" xml:space="preserve">
+<style type="text/css">
```

Comment:
Can you minify the SVG via https://jakearchibald.github.io/svgomg/ ? Looking at the amount of `<g>`-tags I'm hopeful that it is able to make it noticeably smaller 🙂